pkg/config/config.go

```diff
@@ -238,6 +238,7 @@ func buildDefaultConfigMap(cm *corev1.ConfigMap) {
 	cm.Data = map[string]string{
 		routingSuffix:              defaultRoutingSuffix,
 		pluginArtifactsBrokerImage: defaultPluginArtifactsBrokerImage,
+		webhooksEnabled:            defaultNoConfigMapWebhooksEnabled,
```

Comment:
Do we really need defaults on configmap level? Maybe embedded in controller defaults are enough?
Then it would be easier to change defaults if we need it.
Otherwise - we can't check if it's a default in configmap we should be free to change or it's admin set value?

If we talk about webhooks enabled - then it's going to be unmodified I believe.
But if we talk about plugin broker - I believe the default value is going to be changed pretty soon.

So, I would create empty may just for admin easier find to place where put custom configuration but without values.
Maybe only ones which are evaluated depending on the cluster, like routingSuffix on OpenShift Infra.
@amisevsk @JPinkney WDYT?